src/componentManifest.ts

```diff
@@ -13,34 +13,155 @@ import { apiVersions } from './sdkVersion';
 const PLUGIN_NAME = 'componentManifest';
 const manifestPath = 'manifest.json';
 
+/**
+ * Descriptor for localized resources.
+ *
+ * On device, `locale` is capped at 64 bytes,
```

Comment:
Which field does this refer to? It looks like the three below are covered by other comments.

Reply:
I forgot to remove it from here after moving the comment to the actual field. Fixed, thanks.